mln/services/inventory.py

```diff
@@ -35,6 +36,10 @@ def remove_inv_item(user, item_id, qty=1):
 	else:
 		raise RuntimeError("%s has fewer items than the %i requested to delete" % (stack, qty))
 
+def return_invalid_modules(user): 
```

Comment:
Calling the function "return_" is maybe a bit confusing, since it doesn't `return` modules to the caller, just to the user's inventory. Maybe something like `refund`? `restore`?